nextjsUIapp/src/app/data-release/page.tsx

```diff
@@ -115,18 +121,48 @@ export default async function Contact() {
+                            BICAN Rapid Release
                         </td>
                         <td className="px-6 py-4">
-                            NA
+                            -
                         </td>
                         <td className="px-6 py-4">
-                            NA
+                            CSV
+                        </td>
+                        <td className="px-6 py-4">
+                            <a href="/data-release/all_data.csv" download>All Data</a>
+                        </td>
+                    </tr>
+                    <tr className="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
+                        <th scope="row"
+                            className="px-6 py-4 font-medium text-gray-900 whitespace-nowrap dark:text-white">
+                            03 October, 2024
+                        </th>
+                        <td className="px-6 py-4">
+                            BICAN Rapid Release
+                        </td>
+                        <td className="px-6 py-4">
+                            <a href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/Enriched-Cell-Sample-data.csv" download>EnrichedCellSample</a> | <a
+                            href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/Library-data.csv" download>Library</a> | <a
+                            href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/Tissue-Sample-data.csv" download>TissueSample</a> | <a
+                            href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/Brain-Slab-data.csv" download>BrainSlab</a> | <a
+                            href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/AmplifiedCdna.csv.csv" download>AmplifiedCdna</a> | <a
+                            href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/BarcodedCellSample.csv" download>BarcodedCellSample</a>
+                            | <a
+                            href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/DissectionRoiPolygon.csv" download>DissectionRoiPolygon</a> | <a
+                            href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/DissociatedCellSample.csv" download>DissociatedCellSample</a> | <a
+                            href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/LibraryAliquot.csv" download>LibraryAliquot</a> | <a
+                            href="https://brainkb-data-release.s3.us-east-2.amazonaws.com/data-release/bican-rapid-release/10_2024/Donor.csv" download>Donor</a>
```

Comment:
Do you want to make some of these constants so that it is easier to update in singular place in the future without missing an update?

Reply:
Code updated, now the data are dynamically fetched from AWS